freqtrade/commands/arguments.py

```diff
@@ -12,7 +12,7 @@
 
 ARGS_COMMON = ["verbosity", "logfile", "version", "config", "datadir", "user_data_dir"]
 
-ARGS_STRATEGY = ["strategy", "strategy_path"]
+ARGS_STRATEGY = ["strategy", "strategy_path", "recursive_strategy_search"]
```

Comment:
I think this should be added to strategy-list as well (below) - which defines this separately (as it can't take a `strategy` argment).

Reply:
What do you mean exactly? It looks like when looping through the strategy_list in `backtesting.__init__` and `backtesting.start`, `load_strategy` and then `_load_strategy` is called, which already accounts for `recursive_strategy_search`

Reply:
the command where it's missing is `freqtrade list-strategies`. That's a few lines below this one ... just search for `strategy_path` in this file.